ovis_bringup/package.xml

```diff
@@ -1,55 +1,14 @@
 <?xml version="1.0"?>
-<package format="2">
+<package>
   <name>ovis_bringup</name>
   <version>0.0.0</version>
   <description>
       The ovis_bringup package.
-      Contains launch files for interacting with Kinova actuators. 
+      Contains launch files for interacting with Ovis arm. 
   </description>
 
-  <!-- One maintainer tag required, multiple allowed, one person per tag -->
-  <!-- Example:  -->
-  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
-  <maintainer email="ros@kinovarobotics.com">Kinova</maintainer>
-  <!-- One license tag required, multiple allowed, one license per tag -->
-  <!-- Commonly used license strings: -->
-  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <maintainer email="capra@clubcapra.com">Capra</maintainer>
   <license>BSD</license>
-
-
-  <!-- Url tags are optional, but mutiple are allowed, one per tag -->
-  <!-- Optional attribute type can be: website, bugtracker, or repository -->
-  <!-- Example: -->
-  <!-- <url type="website">http://wiki.ros.org/kinova_bringup</url> -->
-
-
-  <!-- Author tags are optional, mutiple are allowed, one per tag -->
-  <!-- Authors do not have to be maintianers, but could be -->
-  <!-- Example: -->
-  <!-- <author email="jane.doe@example.com">Jane Doe</author> -->
-
-
-  <!-- The *_depend tags are used to specify dependencies -->
-  <!-- Dependencies can be catkin packages or system dependencies -->
-  <!-- Examples: -->
-  <!-- Use build_depend for packages you need at compile time: -->
-  <!--   <build_depend>message_generation</build_depend> -->
-  <!-- Use buildtool_depend for build tool packages: -->
-  <!--   <buildtool_depend>catkin</buildtool_depend> -->
-  <!-- Use run_depend for packages you need at runtime: -->
-  <!--   <run_depend>message_runtime</run_depend> -->
-  <!-- Use test_depend for packages you need only for testing: -->
-  <!--   <test_depend>gtest</test_depend> -->
   <buildtool_depend>catkin</buildtool_depend>
-  <exec_depend>kinova_driver</exec_depend>
-  <exec_depend>kinova_msgs</exec_depend>
```

Comment:
Why remove the kinova dependencies? You seem to need them in ovis_bringup/launch/kinova_driver.launch?

Reply:
I've only added the `kinova_driver` to the `package.xml` since we don't call directly `kinova_msgs` in this package.